app/controllers/project_groups_controller.rb

```diff
@@ -32,33 +34,38 @@ def new
   end
 
   def create
-    @project_group = project_group_source.new
-    project_group_source.transaction do
-      @project_group.assign_attributes(name: group_params[:name])
-      filter = ::Filters::HudFilterBase.new(user_id: current_user.id, project_type_numbers: []).update(filter_params.merge(coc_codes: []))
-      filter.coc_codes = []
-      @project_group.options = filter.to_h
-      if @project_group.save!
-        editors = user_params[:editor_ids]&.reject(&:blank?)&.map(&:to_i)
-        # If the user can't edit all project groups, make sure we add the user so they can access it later
-        editors << current_user.id
-
-        @project_group.replace_access(User.find(editors), scope: :editor)
-        @project_group.maintain_projects!
-        Collection.maintain_system_groups(group: :project_groups)
-
-        # TODO: START_ACL remove when ACL transition complete
-        users = user_params[:users]&.reject(&:empty?)&.map(&:to_i)
-        users << current_user.access_group.id
-        @project_group.update_access(users)
-        AccessGroup.maintain_system_groups(group: :project_groups)
-        # END_ACL
+    begin
```

Comment:
This adjustment is mostly to get the tests happy, the UI seemed ok, but I think might not have been if it had raised.  There are two changes here:
1. move the rescue outside of the transaction
2. call `@project_group.instance_variable_set(:@filter, nil)` to force a recalculation of the filter before calling `@project_group.maintain_projects!` (Maybe that should just be part of the method, but it only seemed to be an issue in the controller/tests.